docs/commands/rpc-protocol.md

```diff
@@ -829,10 +829,9 @@ Using the optional `json_block` is recommended since v19.0.
   }
 }
 ```
-**Optional "pending", "source", "balance"**
+**Optional "pending", "source"**
 
 _pending, source: version 9.0+_  
-_balance: version 12.0+_  
```

Comment:
Technically, we should say that balance was optional between v12 and v??

Reply:
balance stopped being optional at v18.0

Reply:
Yeah I think leaving that off makes sense given how long ago V12 was (April 2018 actually, and even V18 is no longer an active release). Also the addition of `balance` in V18 is actually covered above in the summary of the call.